utilities/db_util.py

```diff
@@ -1,36 +1,36 @@
-from dotenv import dotenv_values
+import os
+import logging
 import mysql.connector
 
-def connect_to_db(db: str=""):
-  config = dotenv_values(".env")
+def connect_to_db(db: str = ""):
   try:
     db_config = { 
       'host': 'localhost', 
-      'user': config['USER'], 
-      'password': config['PASSWORD']
+      'user': os.getenv('USER'),
+      'password': os.getenv('PASSWORD')
     } 
     if db: 
       db_config['database'] = db 
     mydb = mysql.connector.connect(**db_config)
     return mydb
-  except (MySQLdb.Error, MySQLdb.Warning) as e:
-    print(e)
+  except Exception as e:
+    logging.error(e)
```

Comment:
`MySQLdb` is not imported so this will snap, and with `Exception` we are covering more cases